src/bicycle/model.py

```diff
@@ -43,8 +107,53 @@ def forward(self, x):
         return mu, variance
 
 
-class Omega_Iterative(pl.LightningModule):
-    def __init__(self, alpha, beta, sigma, lr=1e-2, sigma_min=1e-3, device="cpu"):
+class OmegaIterative(pl.LightningModule):
+    """
+    This class gives a parametrization for omega and computes B.
+    It utilizes continuous lyapunov equation and assumes steady state.
+    The class subsets pyTorch_lightnings LightningModule class.
+
+    Attributes:
+        alpha (torch.Tensor): A tensor representing the alpha parameter,
+            detached and moved to the specified device.
+        beta (torch.Tensor): A tensor representing the beta matrix
+            (gene gene interactions), detached and moved to the specified device.
+        sigma (torch.Tensor): A tensor representing the sigma parameter,
+            detached and moved to the specified device.
+        n_genes (int): The number of genes, derived from the shape of the alpha tensor.
+        lr (float): The learning rate for the optimizer. Default is 1e-2.
+        sigma_min (float): The minimum value for sigma. Default is 1e-3.
+        omega (torch.nn.Parameter): A trainable parameter matrix
+            initialized with random values.
+        B (torch.Tensor): A matrix derived from the beta matrix,
+            used in Lyapunov computations.
+
+    Methods:
+        configure_optimizers():
```

Comment:
Why are these docstrings here instead of directly with the methods below?
Also, the configure_optimizers can be removed, because it's only a single, obvious line of code.